src/query/functions-v2/src/scalars/variant.rs

```diff
@@ -76,42 +69,6 @@ pub fn register(registry: &mut FunctionRegistry) {
         }),
     );
 
-    registry.register_passthrough_nullable_1_arg::<BooleanType, VariantType, _, _>(
-        "parse_json",
-        FunctionProperty::default(),
-        |_| None,
-        vectorize_with_builder_1_arg::<BooleanType, VariantType>(|v, output, _| {
-            if v {
-                output.put_slice(JSONB_TRUE);
-            } else {
-                output.put_slice(JSONB_FALSE);
-            }
-            output.commit_row();
-            Ok(())
-        }),
-    );
-
-    for ty in ALL_NUMERICS_TYPES {
-        with_number_mapped_type!(|NUM_TYPE| match ty {
-            NumberDataType::NUM_TYPE => {
-                registry
-                    .register_passthrough_nullable_1_arg::<NumberType<NUM_TYPE>, VariantType, _, _>(
-                        "parse_json",
-                        FunctionProperty::default(),
-                        |_| None,
-                        vectorize_with_builder_1_arg::<NumberType<NUM_TYPE>, VariantType>(
-                            |v, output, _| {
-                                let value: JsonbValue = v.into();
-                                value.to_vec(&mut output.data);
-                                output.commit_row();
-                                Ok(())
-                            },
-                        ),
-                    );
-            }
-        })
-    }
-
     registry.register_passthrough_nullable_1_arg::<VariantType, VariantType, _, _>(
         "parse_json",
```

Comment:
Can we also remove this one?

Reply:
It depends on whether we need to compatible with `Snowflake`, maybe we need some discussion.

Reply:
We should.